views/index.html.twig

```diff
@@ -1,183 +1,118 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="robots" content="noindex,nofollow" />
-    <title>{{ name }} Composer Repository</title>
+
+    <title>{{ name }} Composer repository</title>
+
     <style type="text/css">
-    {% include 'styles.css' %}
+        {% include 'assets/bootstrap.min.css' %}
     </style>
 </head>
+
 <body>
 
-    <div id="doc" class="yui-t7">
-        <div id="hd">
-            <h1>
-                {% if name %}
-                    {{ name }}
-                {% else %}
-                    The Composer Repository
-                {% endif %}
-            </h1>
-            {% if description %}
-                <p>{{ description|raw }}</p>
-            {% endif %}
-            <div id="lastupdate">
-                Last updated: <time datetime="{{ "now"|date(constant('\DateTime::ISO8601')) }}">{{ "now"|date(constant('\DateTime::RSS')) }}</time>
+<nav class="navbar navbar-default">
+    <div class="container">
+        <form class="navbar-form navbar-right">
+            <div class="form-group">
+                <label for="search">Filter:</label>
+                <input type="text" id="search" class="form-control" autocomplete="off" autofocus />
             </div>
-        </div>
-        <div id="bd">
-            <div class="yui-g">
-                Adding the repository to <em>composer.json</em>:
-                <pre><code>{
+        </form>
+    </div>
+</nav>
+
+<div class="container">
+
+    <div class="page-header">
+        <h1>
+            {% if name %}{{ name }}{% else %}Composer repository{% endif %}
+        </h1>
+
+        <small id="lastupdate">
+            Last updated: <time datetime="{{ "now"|date(constant('\DateTime::ISO8601')) }}">{{ "now"|date(constant('\DateTime::RSS')) }}</time>
```

Comment:
should be updated to ``\DateTime::ATOM``, which is the real ISO format. ``\DateTime::ISO8601`` is not actually compatible with the ISO format